affiliates/mexico.swagger.json

```diff
@@ -416,15 +416,17 @@
           },
           "remuneration_deadline" : {
             "type" : "string",
-            "example" : "yyyy-mm-dd"
+            "example" : "2023-04-18"
```

Comment:
but what's the format?

Reply:
added format description

Reply:
according to json format validator, there is `format` field, why don't we use that?

Reply:
now using "date" format and regex pattern only allows 1000-2999 for year